packages/block-editor/src/utils/pasting.js

```diff
@@ -70,7 +70,16 @@ export function shouldDismissPastedFiles( files, html /*, plainText */ ) {
 		// other elements found, like <figure>, but we assume that the user's
 		// intention is to paste the actual image file.
 		const IMAGE_TAG = /<\s*img\b/gi;
-		return html.match( IMAGE_TAG )?.length !== 1;
+		if ( html.match( IMAGE_TAG )?.length !== 1 ) return true;
```

Comment:
Why does this mean we should dismiss files? Should we check if the length is the same as the amount of files provided?

Reply:
Oh I see the `files?.length === 1` check now.

Reply:
Yeah, the two if-statements combined mean that we can only dismiss files when there is a mismatch between the number of files and the number of IMG tags in the HTML. The first if-statement just reduces the surface area of this helper, making it more conservative (e.g. there haven't been reports of unexpected file pastes when there is more than one file, so might as well only check with `files.length == 1`).